rest-service/manager_rest/manager_exceptions.py

```diff
@@ -24,6 +24,15 @@ def __init__(self, http_code, error_code, *args, **kwargs):
         self.error_code = error_code
 
 
+class InsufficientMemoryError(ManagerException):
+    INSUFFICIENT_MEMORY_ERROR_CODE = 'insufficient_memory_error'
+
+    def __init__(self, *args, **kwargs):
+        super(InsufficientMemoryError, self).__init__(
+            400, InsufficientMemoryError.INSUFFICIENT_MEMORY_ERROR_CODE,
```

Comment:
I think error code 503 would be more suitable here and 4xx are usually for client side errors.